src/components/DeploymentFormV2.vue

```diff
@@ -4,9 +4,16 @@
       General
     </h3>
 
-    <p-label label="Name">
-      <p-text-input :model-value="name" disabled />
-    </p-label>
+    <template v-if="mode === 'duplicate'">
+      <p-label label="Name" :state="nameState" :message="nameError">
+        <p-text-input v-model="name" />
+      </p-label>
+    </template>
+    <template v-else>
+      <p-label label="Name">
+        <p-text-input :model-value="name" disabled />
+      </p-label>
+    </template>
```

Comment:
```suggestion
    <p-label label="Name" :state="nameState" :message="nameError" :disabled="mode === 'update'" >
        <p-text-input v-model="name" />
    </p-label>
```
You can use the same markup here and conditionally disable the input, I think